feg/protos/basic_acct.proto

```diff
@@ -49,16 +49,12 @@ message AcctSession {
         bytes  hardware_addr = 3;
         string name = 4;
     }
-    // ID of the user network (MNO, Identity Provider, etc.)
-    string user_network_id = 5;
```

Comment:
Instead of deleting the tag in a PR that deprecates it's use, opt to either use `string user_network_id = 5 [deprecated=true];` (if you want deprecation to be soft / warning based in compilers) - or remove the line entirely but add the tag to the reserve list:
```
message AcctSession {
    reserved 5,other_tags_if_any;

    oneof user { ...
```

The reserved annotations inform future modifications of the protobuf definition - that client or server code out there in the wild may already have this tag associated with some now-unused data field... and it should be avoided for safety and backwards compatibility.

Reply:
Some more details on the nomenclature and impact are available at https://developers.google.com/protocol-buffers/docs/proto

Reply:
Hey @electronjoe , I think this is an experimental service that hasn't been used yet.

On services that are being used we normally don't remove old values, we just leave them there. But maybe for the future it is a good practice to, at least, use `[deprecated]`

Reply:
We generally know how to preserve backward compatibility of protos, but in this particular case, it's not necessary, this is still a proto definition state, no services or APIs using this proto exist yet. So, there is no reason to pollute the design with unnecessary, depreciated fields.